dbt_copilot_helper/commands/dns.py

```diff
@@ -741,5 +746,212 @@ def assign(app, domain_profile, project_profile, svc, env):
         return
 
 
+@domain.command()
+@click.option(
+    "--project-profile", help="AWS account profile name for certificates account", required=True
+)
+@click.option("--env", help="AWS Copilot environment name", required=True)
+@click.option("--app", help="Application Name", required=True)
+@click.option("--svc", help="Service Name", required=True)
+@click.option("--delete", help="Delete the CDN domain", is_flag=True)
+# Feature to be added at later date if needed, there shouldn't be a need to remove all domains
+# @click.option("--force", help="Force remove", is_flag=True)
+def cdn(project_profile, env, app, svc, delete, force=False):
```

Comment:
This is quite a large method. I think my earlier comment re having the delete command separate would help there, along with extracting common parts to methods for easier grokking.

Reply:
Agreed with having a separate `copilot-helper cdn delete` command

Reply:
added